modules/fedimint-ln-tests/Cargo.toml

```diff
@@ -27,4 +24,14 @@ fedimint-client = { path = "../../fedimint-client" }
 fedimint-core ={ path = "../../fedimint-core" }
 fedimint-server = { path = "../../fedimint-server" }
 fedimint-logging = { path = "../../fedimint-logging" }
+lightning-invoice = { version = "0.21.0", features = [ "serde" ] }
 tokio = { version = "1.26.0", features = ["sync"] }
+tracing = "0.1.37"
+# Remove once we modularize the gw / ln server
+fedimint-wallet-common = { path = "../fedimint-wallet-common" }
+fedimint-wallet-client = { path = "../fedimint-wallet-client" }
+fedimint-wallet-server = { path = "../fedimint-wallet-server", features = ["native"] }
+fedimint-wallet-tests = { path = "../fedimint-wallet-tests" }
+fedimint-mint-common = { path = "../fedimint-mint-common" }
+fedimint-mint-client = { path = "../fedimint-mint-client" }
+fedimint-mint-server = { path = "../fedimint-mint-server" }
```

Comment:
Then these would be replaced with `brrrrtimint` aka the dummy module?

Reply:
Yes, ideally we will only test modules independently with the dummy to demonstrate they have no dependencies on other modules.

The tests in the gateway crate will probably need to use the wallet and ln modules though to test the deposit RPC.